tests/ui/empty_loop_no_std.rs

```diff
@@ -10,11 +10,15 @@ use core::panic::PanicInfo;
 
 #[start]
 fn main(argc: isize, argv: *const *const u8) -> isize {
-    loop {}
+    // This loop should not cause a warning.
+    loop {
+        core::sync::atomic::spin_loop_hint();
```

Comment:
Should this be updated to match one of the current suggestions (`read_volatile`/`llvm_asm!`)?

Reply:
I'll change it to `read_volatile`.